content/en/tracing/profiling/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Profiling
+kind: Documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+<div class="alert alert-info">
+Datadog Profiling is in beta. Reach out to <a href="/help/">Datadog Support</a> if you encounter any issues or have feedback to share.
+</div>
+
+Profiling libraries are shipped within the following tracing language library. Refer to the dedicated language page to learn how to enable profile connections for your application:
+
+{{< partial name="profiling/profiling-languages.html" >}}
+
+## Profile explorer
+
+Once your profile collection is set up, profiles are available in the [APM > Profiling][1] page within Datadog:
+
+{{< img src="tracing/profiling/profiling_main_page.png" alt="Profiling main page">}}
+
+Each line represents a profile for a process on which Datadog profiling was running for a short amount of time. Depending on the language, these processes are profiled between 15s and 60s.
+
+### Search
+
+You can filter according to infrastructure tags or application tags set up from your [environment tracing configuration][2]. By default the following facets are available:
+
+|  Facet   |  Definition                                                             |
+| -------- | ----------------------------------------------------------------------- |
+| Env      | The environment your application is running on (`prod`, `staging`).     |
+| Service  | The name of the [service][3] your code is running.                      |
+| Version  | The version of your code.                                               |
+| Host     | The host name your profiled process is running on.                      |
+| Runtime  | The type of runtime the profiled process is running (`JVM`, `CPython`). |
+| Language | The language of your code (`Java`, `Python`).                           |
```

Comment:
Language and runtime still need some love. IMO we should leave them out of the docs until then. 

Maybe even then; if this is a getting-started-type doc (rather than reference material), we don't want to overwhelm new users with details. They'll see what facets we have by default. Maybe that even argues for deleting this section altogether.

Before we do that though, let's talk through the high level structure. What types of docs do we have/want? Quick start? Into tutorial? How tos? Reference material? For one example, see how gcp differentiates these- 
![image](https://user-images.githubusercontent.com/16651846/75443092-b51a7880-592e-11ea-801a-f085c08ea919.png)

Reply:
Let's keep it for now as it allows a better discoverability of the profiling product if a user is not a Datadog customer.

For now the whole content can fit in one page, hence the current layout, as soon as we have material for guides, Getting started or more reference material we will update the layout of this section accordingly.

Reply:
I'm all about discoverability! I'm not sure I follow though, how does it hurt discoverability if we have a quickstart for power users that are already familiar with profilers (in general, not ours), and also a longer tutorial for less experienced users?

Reply:
Either way, this is still relevant:

Language and runtime still need some love; for the moment they're not always set, nor always have exactly what they say they have. In fact `host` is similar: on vms it's actually the host. In k8s (for now) it's actually the name of the pod. IMO while they're ~wrong, we should leave them out of the docs.

Again, leaving something (obvious) out of the docs seems worse to me than including something more or less incorrect in the docs.

Reply:
Nothing is obvious, otherwise there would be no need for a documentation in a first place.  I updated the host description, let's address other confusion in further PR.